imap_processing/hit/l0/hit_apid.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class HitAPID(Enum):
```

Comment:
```suggestion
class HitAPID(IntEnum):
```
Probably not a huge deal, but if you do want to verify against numbers later it might help to be explicit that these are ints?

```python
assert HitAPID.HIT_AUT == 1250
```

Reply:
Ah cool, I didn't know about IntEnum